linux/Libav.cpp

```diff
@@ -776,7 +777,13 @@ void CodecLibAV::StreamInitialise()
                                      iTrackLengthJiffies,
                                      0,
                                      false,
+<<<<<<< HEAD
```

Comment:
You're leaving conflict markers in here

Reply:
Hi @DoomHammer, 
thanks for the hind. I corrected this. 
Br André